bin.js

```diff
@@ -36,9 +36,16 @@ log.info('begin', 'Prebuild-install version', prebuildClientVersion)
 
 var opts = Object.assign({}, rc, {pkg: pkg, log: log})
 
-var execPath = process.env.npm_execpath || process.env.NPM_CLI_JS
+var isNpm = true
 
-if (util.isYarnPath(execPath) && /node_modules/.test(process.cwd())) {
+try {
+  var pm = whichPmRuns()
```

Comment:
If `whichPmRuns()` threw an error instead of returning `undefined` this could be

```js
try {
  isNpm = whichPmRuns().name === 'npm'
} catch (err) { }
```

@zkochan WDYT?

Reply:
Actually, I checked `which-pm-runs` and it won't ever fail, so I simplified the code by removing the redundant try/catch